amundsen_application/static/js/components/common/ResourceStatusMarker/styles.scss

```diff
@@ -0,0 +1,49 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+@import 'typography-default';
+
+.failure-icon,
+.success-icon {
+    width: $icon-size;
+    height: $icon-size;
+    border-radius: 100%;
+    display: inline-block;
+    vertical-align: middle;
+}
+.failure-icon {
+    background-color: $icon-status-missed;
+    .exclamation-top {
+        width: 2px;
+        height: 4px;
+        background-color: $icon-status-color;
+        margin: 4px 6px 3px;
+    }
+    .exclamation-bottom {
+        width: 2px;
+        height: 2px;
+        border-radius: 100%;
+        background-color: $icon-status-color;
+        margin: -2px 6px;
+    }
+}
+
+.success-icon{
+    background-color: $icon-status-hit;
+    .icon-check {
+        background-color: $icon-status-color;
+        width: 10px;
+        height: 10px;
+        margin-top: -10px;
+        margin-left: 2px;
+        min-width: 0px;
+    }
+}
+
+.status-text {
+    display: inline-block;
+    margin-left: $spacer-1;
+    vertical-align: middle;
+    @extend %text-body-w2;
```

Comment:
I would expect this to give you a warning, as they usually need to go at the top with a space below.